packages/module/src/ContentHeader/ContentHeader.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { Divider, Flex, FlexItem, Icon, Label, Split, SplitItem, Text } from '@patternfly/react-core';
+import { ExternalLinkAltIcon } from '@patternfly/react-icons';
+import ActionMenu, { ActionMenuProps, ActionMenuVariant } from '../ActionMenu/ActionMenu';
+
+export interface ContentHeaderProps {
+/** title for content header */ 
+  title: string;
+/** subtitle for content header */
+  subtitle: string;
+/** Optional link below subtitle */
+  link?: string;
```

Comment:
I've proposed some changes of this prop below to make sure it follows some standards on the one hand, but still remains customizable.